appium-dotnet-driver/Appium/Windows/WindowsDriver.cs

```diff
@@ -0,0 +1,197 @@
+        /// <param name="commandTimeout">The maximum amount of time to wait for each command.</param>
+        public WindowsDriver(Uri remoteAddress, DesiredCapabilities desiredCapabilities, TimeSpan commandTimeout)
+            : base(remoteAddress, SetPlatformToCapabilities(desiredCapabilities, Platform), commandTimeout)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the WindowsDriver class using the specified Appium local service, desired capabilities, and command timeout.
+        /// </summary>
+        /// <param name="service">the specified Appium local service</param>
+        /// <param name="desiredCapabilities">An <see cref="ICapabilities"/> object containing the desired capabilities.</param>
+        /// <param name="commandTimeout">The maximum amount of time to wait for each command.</param>
+        public WindowsDriver(AppiumLocalService service, DesiredCapabilities desiredCapabilities, TimeSpan commandTimeout)
+            : base(service, SetPlatformToCapabilities(desiredCapabilities, Platform), commandTimeout)
+        {
+        }
+
+        #region IFindByWindowsUIAutomation Members
+
+        /// <summary>
+        /// Finds the first element that matches the Windows UIAutomation selector
+        /// </summary>
+        /// <param name="selector">UIAutomation selector</param>
+        /// <returns>First element found</returns>
+        public W FindElementByWindowsUIAutomation(string selector)
+        {
+            //return (W)this.FindElement("-windows uiautomation", selector);
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Finds a list of elements that match the Windows UIAutomation selector
+        /// </summary>
+        /// <param name="selector">UIAutomation selector</param>
+        /// <returns>ReadOnlyCollection of elements found</returns>
+        public ReadOnlyCollection<W> FindElementsByWindowsUIAutomation(string selector)
+        {
+            //return CollectionConverterUnility.ConvertToExtendedWebElementCollection<W>(
+            //    this.FindElements("-windows uiautomation", selector));
+            throw new NotImplementedException();
+        }
+
+        #endregion IFindByWindowsUIAutomation Members
+
+        /// <summary>
+        /// Hides the keyboard
+        /// </summary>
+        /// <param name="key"></param>
+        /// <param name="strategy"></param>
+        public new void HideKeyboard(string key, string strategy = null)
+        {
+            base.HideKeyboard(strategy, key);
+        }
+
+        /// <summary>
+        /// Create a Windows Element
+        /// </summary>
+        /// <param name="elementId">element to create</param>
+        /// <returns>WindowsElement</returns>
+        protected override RemoteWebElement CreateElement(string elementId)
+        {
+            return new WindowsElement(this, elementId);
+        }
+
+        public override W ScrollTo(string text)
```

Comment:
[There is the commit](https://github.com/appium/appium-dotnet-driver/commit/6c3dddf1bdb184a4524069c50e9ffe1a0e815103). Obsolete code was removed.
Please remove this method.